employee-benefits/professional-development.md

```diff
@@ -27,45 +26,45 @@ Historical Prodev Hours Spent:
 | 2018 | 1736                           | 44                            | 39.5                                          |
 | 2019 | 1668                           | 56                            | 29.8                                          |
 
-_\*Snapshot from February 12, 2020._
+_\*Snapshot from February 12, 2020._</details>
```

Comment:
@ContentBrewmaster looks like the details displays works but the table inside of it is not rendered. Might just need some tweaking to make it work. See preview https://civicactions-handbook--1036.org.readthedocs.build/en/1036/employee-benefits/professional-development/

Reply:
Good question. The table of the practice area does work https://civicactions-handbook--1036.org.readthedocs.build/en/1036/practice-areas/about-practice-areas/. Might need some more R&D.

Reply:
Looks like we need another plugins from https://github.com/squidfunk/mkdocs-material/issues/4084 called 'Markdown in HTML' see https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown/#markdown-in-html. I will push up that change.